src/main/java/uk/gov/hmcts/reform/blobrouter/reconciliation/service/ReconciliationMailService.java

```diff
@@ -164,7 +164,7 @@ private void sendMailNoSupplierStatement(LocalDate date, TargetStorageAccount ac
         try {
             emailSender.sendMessageWithAttachments(
                 account.name() + " Scanning Reconciliation NO REPORT RECEIVED",
-                "No Report received for " + date,
+                "No supplier statement received for " + date,
```

Comment:
should email title be changes as well then?

Reply:
it was like that in the ticket.
but I think we can change

Reply:
yeah, it would be clearer to stick to same naming convention